### PR TITLE
updated FAQ

### PR DESCRIPTION
**Pull request by HimanshuRanka**
## Briefly describe the feature or bug that this PR addresses.

- updated "where do we get data from"
- added "how often is data updated"

## Please link the Airtable ticket associated with this PR.

https://airtable.com/appT8tnLbGJV6v81V/tbli2lWQHAqBa6ZcI/viw2Kjofxz0J1dHD0/rec9w4yKrsZqx4QnJ?blocks=hide

## Does this PR require any French translations? Have they been included?
Yes
### Explore

- Check Desktop and on Mobile
- Summary Stats
- Map loads in ~8 seconds with loading spinner 
- Map has pins
- Country Modal - check all fields
- Pins modal (check each grade) - check all fields
- Filter info bubbles
- Select each filter and check options
- Execute a Filter for Risk of bias, Sex, and Date.
- Check the "Database up to date to:" date
- Check each footer item (Privacy policy etc.)

### Analyse
- Check tableau embed loads and is functional
- Check for double scrolling (the embed scrolls within the page) 

### Data
- Click each button
- Click a FAQ for functionality
- References table embed loads
    - Check that downloads are disabled.
- Data download link provides access to a downloadable CSV.
    - Download the CSV
- "terms of use" button

### Publications
- Cards all load with images
- Carousel is clickable
- Try 2 random items on the page for functional links

### About
- Page loads
- Scroll to bottom

### Check serotracker.com/broken
- 404 page loads and animation works

### Check serotracker.com/Canada 
- Check tableau embed loads and is functional
- Check for double scrolling (the embed scrolls within the page) 
- Check in French

### CODE REVIEW

File: src/utils/translate/en.json [hunk 1]
```diff
@@ -145,7 +145,16 @@
   "DropdownQuestions": {
     "Question1": {
       "Question": "Where does SeroTracker's data come from?",
-      "Answer": "We collect data from published peer reviewed research articles, preprints, reports, and media (unpublished grey literature). The sources included in our dashboard to date are in the table below."
+      "Answer": {
+        "Part1": "We collect all seroprevalence data from published peer reviewed research articles, preprints, reports, and media (unpublished grey literature). Data related to vaccination status and case counts is pulled from the",
+        "Part2": "The sources included in our dashboard to date are in the table below."
+      },
+      "Links": {
```
Comment (atatmaja) [inline src/utils/translate/en.json:152]: Nit: this doesn't need to be in a translations file

**Comment by atatmaja** [comment]
We also need german translations now. @simonarocco what was our plan to collaborate with the RKI to maintain this?

**Review by atatmaja — CHANGES_REQUESTED** [review]
Requesting changes b/c no german translations

File: src/utils/translate/en.json (hunk 1 above)
Comment (HimanshuRanka) [reply src/utils/translate/en.json:152]: wait which part doesn't need to be in translations file?

Comment (atatmaja) [reply src/utils/translate/en.json:152]: The links, as URLs are language agnostic

Comment (HimanshuRanka) [reply src/utils/translate/en.json:151]: I think having links here is actually helpful. Looking at the translate function for the FAQ questions the links are in the middle of answers, so either we have links there or we hardcode the links with multiple translate functions in the code file. I think either works.

File: .gitignore [hunk 2]
```diff
@@ -35,3 +35,4 @@ load-dotenv.sh
 
 .idea/
 
+get_diff.py
```
Comment (atatmaja) [inline .gitignore:38]: Why gitignore this? We probably want everyone on the team being able to use this right?

**Review by atatmaja — APPROVED** [review]
LGTM

**Pull request merged**